src/pages/[page].tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
+import fs from "fs";
+import dynamic from "next/dynamic";
+import React from "react";
+
+/*
+// This is a dynamic page to generate pages based on the routes.json file
+// For each theme we must define the list of routes and supply a contentPath
+// The content will be statically generated at build time
+*/
+
+type TPage = {
+  title: string;
+  path: string;
+  contentPath: string;
+};
+
+type TProps = {
+  page?: TPage & {
+    notFound?: boolean;
+  };
+};
+
+export default function Page({ page }: TProps) {
+  // TODO: fix this properly
+  // Next is throwing a NEXT_REDIRECT error under the hood when attemping to navigate to a missing page at root, e.g. /missing-page
+  if (!page || page.notFound) {
+    return window.location.replace("/not-found");
```

Comment:
Do we want to console.error something here?

Reply:
I've added a `console.warn` for now